src/edu/csus/ecs/pc2/core/scoring/DefaultScoringAlgorithm.java

```diff
@@ -918,18 +1023,23 @@ boolean isTeamTied(StandingsRecord standingsRecord, long numSolved, long score,
      *
      * @param problems
      * @param problemsIndexHash
-     * @param summaryMememento
+     * @param summaryMemento
      */
 
-    private void generateSummaryTotalsForProblem(Problem[] problems, Hashtable<ElementId, Integer> problemsIndexHash, IMemento summaryMememento) {
+    private void generateSummaryTotalsForProblem(Problem[] problems, Hashtable<ElementId, Integer> problemsIndexHash, IMemento summaryMemento) {
 
+        //TODO: should we be excluding problems marked as "not active"?
         for (int i = 0; i < problems.length; i++) {
             int id = i + 1;
             problemsIndexHash.put(problems[i].getElementId(), new Integer(id));
-            IMemento problemMemento = summaryMememento.createChild("problem");
-            problemMemento.putInteger("id", id);
+            IMemento problemMemento = summaryMemento.createChild("problem");
```

Comment:
I think all this code is wrong and the original should have been left.  internalId should not be in the problemMemento.  It causes errors in the Junits, and, it does not appear to be used anymore.